app/angular/src/server/config.js

```diff
@@ -15,6 +16,9 @@ export default function(configType, baseConfig, configDir) {
   const babelConfig = loadBabelConfig(configDir);
   config.module.rules[0].query = babelConfig;
 
+  const tsOptions = loadTsConfig(configDir);
+  config.module.rules[1].loaders[0].options = tsOptions;
```

Comment:
Sorry if it looks ugly, IMO need some work moving webpack configurations to the core.